policy/modules/system/logging.te

```diff
@@ -5,6 +5,14 @@ policy_module(logging)
 # Declarations
 #
 
+## <desc>
+## <p>
+## Allows syslogd internet domain sockets
+## functionality (dangerous).
+## </p>
+## </desc>
+gen_tunable(syslog_can_network, false)
```

Comment:
The preference now is that the module name is the first work of the tunable, like it is with interface names. Please name it something like `logging_syslog_networking`.